bundle_install_process.go

```diff
@@ -42,6 +44,15 @@ func (ip BundleInstallProcess) Execute(workingDir, gemLayersDir string) error {
 	buffer = bytes.NewBuffer(nil)
 	args = []string{"install"}
 
+	_, err = os.Stat(filepath.Join(workingDir, "vendor", "cache"))
```

Comment:
Can we also check the value of `bundle config cache_path --parseable` to see if the cache is not in the default location?

According to [these docs](https://bundler.io/v2.1/man/bundle-config.1.html#LIST-OF-AVAILABLE-KEYS), there is a `cache_path` field that a user can override if they want the cache to be in a non-default location. When we look for the cache directory on the filesystem, we should first see if it is configured to be in a non-default location.